next.config.js

```diff
@@ -4,7 +4,7 @@
 
 const nextConfig = {
   reactStrictMode: true,
-
+  trailingSlash: true,
```

Comment:
i know this might be needed in the implementation

i just wanna learn how it works, could you explain?

Reply:
Ofcourse, is boolean flag that generate trailing slashes, in general when we work with subpaths Next js generate they ('/blog/2023'), when turn on this property next js generate ('/blog/2023/'), in this case it was turn on for that, because the library uses there, but It also helps in other things like SEO.